Jamulus.pro

```diff
@@ -1144,6 +1144,6 @@ contains(CONFIG, "disable_version_check") {
 # be sure to update .github/workflows/coding-style-check.yml and .clang-format-ignore as well.
 CLANG_FORMAT_SOURCES = $$files(*.cpp, true) $$files(*.mm, true) $$files(*.h, true)
 CLANG_FORMAT_SOURCES = $$find(CLANG_FORMAT_SOURCES, ^\(android|ios|mac|linux|src|windows\)/)
-CLANG_FORMAT_SOURCES ~= s!^\(windows/nsProcess/|libs/ASIOSDK2/|src/res/qrc_resources\.cpp\)\S*$!!g
+CLANG_FORMAT_SOURCES ~= s!^\(libs/.*/|src/res/qrc_resources\.cpp\)\S*$!!g
```

Comment:
... and thinking about it further... does line 1146 filter out or filter in?  If it's filtering out anything that doesn't match that pattern, `libs/` shouldn't match anyway....

Reply:
I think it should be a filter out?

@hoffie ?

Reply:
[`$$find()`](https://doc.qt.io/qt-5/qmake-function-reference.html#find-variablename-substr) keeps all entries which match the pattern (similar to grep). So, yes, seems like having `libs/` in 1147 is redundant (`src/res/...` isn't, though). Should probably be tested though (run `qmake` with the change, `grep clang-format Makefile`).

Reply:
```
$:~/Proj/jamulus$ make clang_format 
clang-format -i android/androiddebug.cpp src/analyzerconsole.cpp src/audiomixerboard.cpp src/buffer.cpp src/channel.cpp src/chatdlg.cpp src/client.cpp src/clientdlg.cpp src/clientrpc.cpp src/clientsettingsdlg.cpp src/connectdlg.cpp src/levelmeter.cpp src/main.cpp src/multicolorled.cpp src/protocol.cpp src/rpcserver.cpp src/server.cpp src/serverdlg.cpp src/serverlist.cpp src/serverlogging.cpp src/serverrpc.cpp src/settings.cpp src/signalhandler.cpp src/socket.cpp src/util.cpp src/recorder/creaperproject.cpp src/recorder/cwavestream.cpp src/recorder/jamcontroller.cpp src/recorder/jamrecorder.cpp src/sound/soundbase.cpp src/sound/asio/sound.cpp src/sound/coreaudio-mac/sound.cpp src/sound/jack/sound.cpp src/sound/oboe/sound.cpp ios/ios_app_delegate.mm mac/activity.mm mac/badgelabel.mm src/sound/coreaudio-ios/sound.mm ios/ios_app_delegate.h mac/activity.h mac/badgelabel.h src/analyzerconsole.h src/audiomixerboard.h src/buffer.h src/channel.h src/chatdlg.h src/client.h src/clientdlg.h src/clientrpc.h src/clientsettingsdlg.h src/connectdlg.h src/global.h src/levelmeter.h src/multicolorled.h src/protocol.h src/rpcserver.h src/server.h src/serverdlg.h src/serverlist.h src/serverlogging.h src/serverrpc.h src/settings.h src/signalhandler.h src/socket.h src/testbench.h src/threadpool.h src/util.h src/recorder/creaperproject.h src/recorder/cwavestream.h src/recorder/jamcontroller.h src/recorder/jamrecorder.h src/sound/soundbase.h src/sound/asio/sound.h src/sound/coreaudio-ios/sound.h src/sound/coreaudio-mac/sound.h src/sound/jack/sound.h src/sound/oboe/sound.h
```

seems ok?

Reply:
Yep.